guides/source/active_record_validations.md

```diff
@@ -66,8 +66,8 @@ controller-level validations. Here's a summary of the pros and cons:
   users with immediate feedback as they use your site.
 * Controller-level validations can be tempting to use, but often become
   unwieldy and difficult to test and maintain. Whenever possible, it's a good
-  idea to keep your controllers skinny, as it will make your application a
-  pleasure to work with in the long run.
+  idea to keep your controllers as simple as possible, as it will make your
+  application a pleasure to work with in the long run.
```

Comment:
Maybe we could avoid some word repetition by removing the "as possible", given we already say "Whenever possible".
```suggestion
  idea to keep your controllers simple, as it will make your
  application a pleasure to work with in the long run.
```